src/screens/HomeScreen/HomeScreen.js

```diff
@@ -1,19 +1,101 @@
 import React from 'react';
-import { View, Text, SafeAreaView } from 'react-native';
-import styles from './styles';
-import HeaderBar from '../../components/HeaderBar/HeaderBar';
+import { firebase } from '../../config/config';
+import { View, Text, SafeAreaView, Image, TouchableOpacity, FlatList } from 'react-native';
+import { withNavigation } from 'react-navigation';
 import { LOGO } from '../../config/styles.js';
+import styles from './styles';
+
+class HomeScreen extends React.Component{
+    state = {querydetail : []};
+    componentDidMount() {
+        const {querydetail} = this.state
+        firebase.firestore().collection("queries").orderBy('postdatetime', 'desc').onSnapshot((querySnapshot) => {
+            var newentity = []
+            var queryArray = []
+            querySnapshot.forEach((doc) => {
+                q_data = {
+                    "uid": doc.data().id,
+                    "QueryInput": doc.data().QueryInput,
+                    "postTime": doc.data().postTime,
+                    "id": doc.id,
+                    "postdatetime": doc.data().postdatetime,
+                    "answerpresent": doc.data().answer_present,
+                    "queryimage" : doc.data().query_image
```

Comment:
As I said, if you are using lower case _ method, use it everywhere. Make that as a standard for this project. Code quality is a must when it comes to OSS.